.gitignore

```diff
@@ -112,3 +112,6 @@ package-lock.json
 .goutputstream*
 
 
+fakecoursedir/
```

Comment:
Are these changes needed for this PR? They don't seem to actually appear in the other changes.

Reply:
Removed.